src/test/ui/attributes/nonterminal-expansion.rs

```diff
@@ -3,7 +3,8 @@
 macro_rules! pass_nonterminal {
     ($n:expr) => {
         #[repr(align($n))]
-        //~^ ERROR expected unsuffixed literal or identifier, found `n!()`
+        //~^ ERROR expected unsuffixed literal or identifier, found `/*start of expr expansion*/`
+        //~| ERROR incorrect `repr(align)` attribute format
         //~| ERROR incorrect `repr(align)` attribute format
```

Comment:
Not sure why is this error reported twice.
Looks like the parsing is somehow run twice when we are attempting to parse `align($n)` into a `MetaItem`? That shouldn't happen.

Reply:
Something weird is happening here. If I just run `rustc src/test/ui/attributes/nonterminal-expansion.rs`, then the error is not repeated. But when run within the test harness, the error *is* repeated.

Reply:
rustc deduplicates errors, but that’s turned off in the test harness

Reply:
Fair enough, thanks, I'll have to work out what's going on there.